test/tests/benchmark/ansible/group_vars/benchmark

```diff
@@ -0,0 +1,3 @@
+test_machine_log_path: "{{ lookup('env', 'HOME') }}/benchmark/20160304-045310"
```

Comment:
This timestamp is written to this file at runtime when starting to run the test group. Test cases within this group will use this variable afterwards and write logs to this specific path. So this variable changes every time the group runs. I can find a way to restore this variable after test group ends, but I don't think it is necessary.